3001-4000/LIT3327Melchiz2.xml

```diff
@@ -40,7 +56,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <abstract><p>The homily on Melchizedek's origin attributed to <persName ref="PRS3255CyrilI"/>, mainly circulate as part of the theological collection 'Qərəllos'. It was certainly translated from the Greek: the original is lost and the Ethiopic version is the only existing one. 
```

Comment:
I suggest: "One of the two homilies on Melchizedek's origin ..."

"circulate" or "circulates" (depending on the clause before).